tracekit.js

```diff
@@ -248,6 +262,38 @@ TraceKit.report = (function reportModuleWrapper() {
         _onErrorHandlerInstalled = true;
     }
 
+    /**
+     * Uninstall the global onerror handler
+     * @memberof TraceKit.report
+     */
+    function uninstallGlobalHandler() {
+        window.onerror = _oldOnerrorHandler;
```

Comment:
We should check to see if the handler is installed first, otherwise we could be setting someone else's handler to undefined.